engine/storage/configdrive/src/org/apache/cloudstack/storage/configdrive/ConfigDrive.java

```diff
@@ -26,10 +26,11 @@
     public static final String openStackConfigDriveName = "/openstack/latest/";
 
     /**
-     * This is the path to iso file relative to mount point
-     * @return config drive iso file path
+     * Created the path to ISO file relative to mount point.
```

Comment:
What do you think of `Creates the path to ...` instead of `Created`?
Change `formatt` to `format`

Reply:
Thanks. Fixed!